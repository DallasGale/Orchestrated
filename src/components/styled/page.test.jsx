import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import Page from './page'

describe('<Page />', () => {
  test('it renders', () => {
    const Component = render(<Page />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
