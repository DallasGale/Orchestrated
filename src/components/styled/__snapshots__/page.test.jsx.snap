// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> it renders 1`] = `
<main
  class="sc-bwzfXH jSSjEM"
>
  <div
    class="sc-bdVaJa dqmyKm"
  />
</main>
`;
